bin/format_source

```diff
@@ -21,7 +21,10 @@ echo "installed version of clang-format to be used."
 echo ""
 
 find . \( -type d -path ./config -prune -and -not -path ./config \) \
-    -and \( -iname *.h -or -iname *.c -or -iname *.java \) \) \
+    -or \( \( \! \( \
+        -name ncgenyy.c \
```

Comment:
is ncgenyy.c still needed?

Reply:
No. I just wanted to undo my screwup. I'll pull the ncgen bit later.